spyder/plugins/console/widgets/main_widget.py

```diff
@@ -323,17 +326,75 @@ def set_help(self, help_plugin):
         """
         self.shell.help = help_plugin
 
-    def handle_exception(self, text, is_traceback, is_pyls_error=False,
-                         is_faulthandler_report=False):
+    @Slot(dict)
+    def handle_exception(self, error_data, sender=None, internal_plugins=None):
         """
         Exception ocurred in the internal console.
 
         Show a QDialog or the internal console to warn the user.
-        """
+
+        Handle any exception that occurs during Spyder usage.
+
+        Parameters
+        ----------
+        error_data: dict
+            The dictionary containing error data. The expected keys are:
+            >>> error_data= {
+                "text": str,
+                "is_traceback": bool,
+                "repo": str,
+                "title": str,
+                "label": str,
+                "steps": str,
+            }
+        sender: spyder.api.plugins.SpyderPluginV2, optional
+            The sender plugin. Default is None.
+
+        Notes
+        -----
+        The `is_traceback` indicates if `text` contains, plain text or a
+        python error traceback.
+
+        The `title` and `repo` indicate how the error dialog should customize
+        the report dialog and github error submission.
+
+        The `label` and `steps` allow customizing the content of the error
+        dialog.
```

Comment:
Please apply here the same changes I left above.

Reply:
Fixed